error: ‘libintl_setlocale’ is not a member of ‘std’
(This should probably be fixed in <libintl.h>)

--- origsrc/pdf2djvu-0.7.13/i18n.hh	2012-06-03 15:04:41.000000000 -0500
+++ src/pdf2djvu-0.7.13/i18n.hh	2012-08-20 23:59:31.801708800 -0500
@@ -25,6 +25,7 @@ static inline void N_(const char *messag
 #ifdef ENABLE_NLS
 
 #include <libintl.h>
+#undef setlocale
 
 static inline const char * _(const char *message_id)
 {
